gradle.properties

```diff
@@ -12,6 +12,6 @@
 # org.gradle.parallel=true
 #Fri Feb 16 12:00:34 MSK 2024
 org.gradle.parallel=true
-org.gradle.jvmargs=-Xmx3096M -Dkotlin.daemon.jvm.options\="-Xmx3096M"
+org.gradle.jvmargs=-Xmx4096M -Dkotlin.daemon.jvm.options\="-Xmx4096M"
```

Comment:
In some cases assembleDebugAndroidTest resulted in OOM, so I expanded the available memory for gradle